inference-engine/tests/functional/plugin/conformance/test_runner/api_conformance_runner/skip_configs/skip_config_cpu.lst

```diff
@@ -0,0 +1 @@
+.*Auto.*InferRequestPreprocessTest.*SetScalePreProcessSetBlob.*
```

Comment:
do we have a bug? does it work on MULTI, but fails on AUTO?

Reply:
Will be removed